public-ui/src/app/data-browser/views/survey-view/survey-view.component.html

```diff
@@ -201,15 +201,17 @@ <h2 class="secondary-display">
                     <div class="survey-row-expansion"
                       *ngIf="a.expanded && a.stratum4.toLowerCase() != 'did not answer' && a.stratum7 !== '1'"
                       #chartElement>
-                      <app-survey-chart [isCopeSurvey]="isCopeSurvey" [graphButtons]="graphButtons" [question]="q"
-                        [selectedAnalysis]="q.selectedAnalysis" [selectedResult]="a" [surveyName]="survey.name" [versionAnalysis]='surveyVersions'
-                        [surveyCountAnalysis]="surveyCountAnalysis" [searchTerm]="prevSearchText" [answer]="a">
-                      </app-survey-chart>
+                      <app-survey-chart-react [graphButtons]="graphButtons" [isCopeSurvey]="isCopeSurvey" [question]="q" [answer]="a"
+                                              [selectedAnalysis]="q.selectedAnalysis" [selectedResult]="a"
+                                              [surveyName]="survey.name" [versionAnalysis]='surveyVersions'
+                                              [surveyCountAnalysis]="surveyCountAnalysis" [searchTerm]="prevSearchText"
+                                              (click)="$event.stopPropagation();"></app-survey-chart-react>
```

Comment:
Had to have this in here to stop closing the survey answer tab.

Reply:
I may have found a temp workaround so we don't need this. Try adding an optional event param to `showAnswerGraphs` at line 520 in `survey-view.component.ts`, then we can disregard clicks from within the expanded area:
```
public showAnswerGraphs(a: any, q: any, level?: number, event?: MouseEvent) {
    if (event && this.chartEl && this.chartEl.nativeElement.contains(event.target)) {
      return;
    }
```

Then we just need to pass the click event at line 140 in this file:
```
<div [ngClass]="a.expanded?'survey-tbl-r-state':''" class="survey-tbl-exp-r "
                    *ngFor="let a of q.countAnalysis.results" (click)="showAnswerGraphs(a,q,1, $event)"
                    [hidden]="!showAnswer[q.conceptId]">
```
Seems to be working for me, if you could test it to verify it works as intended.